features/FxA-57-verify-reminder/README.md

```diff
@@ -0,0 +1,44 @@
+
+Verification Reminder Email Specification
+======================
+
+https://mozilla.aha.io/features/FXA-57
+
+As a new Firefox Accounts user,
+ I have signed up for an FxA account when **setting up sync**, but did not verify my email.
+ 
+ I should get 2 email reminders that will ask me to verify my email.
+ 
+ The first email arrives after 48 hours since sign up.
+ 
+ The second email arrives after 168 hours since sign up.
+ 
+## Basic Email Template
+
+> This is just a visual guide, please see detailed email content below
+
+![](basic-template.png)
+
+
+## Email 1
+
+### Header: Hello again.
+
+### Content:
+
+A few days ago you created a Firefox Account, but never verified it.
+Once verified, you can access your tabs, bookmarks, passwords and
+ history on any device connected to your Firefox Account.
+ Simply activate your account by confirming this email address.
+
+
+## Email 2
+
+### Header: Still there?
+
+### Content:
+
+A week ago you created a Firefox Account, but never verified it. We’re worried about you.
+Firefox is available for Windows, Mac, Linux, Android and iOS devices.
+You can sync your Firefox passwords, bookmarks and history across all of them.
+Let us know you’re okay by activating this account.
```

Comment:
Confirm this email address to activate your account and let us know you're okay.